LogicalTypes.md

```diff
@@ -255,6 +255,16 @@ The primitive type is a 2-byte fixed length binary.
 
 The sort order for `FLOAT16` is signed (with special handling of NANs and signed zeros); it uses the same [logic](https://github.com/apache/parquet-format#sort-order) as `FLOAT` and `DOUBLE`.
 
+### FIXED_SIZE_LIST
+
+The `FIXED_SIZE_LIST` annotation represents a fixed-size list of elements
+of a primitive data type. It must annotate a `binary` primitive type.
+
+The `fixed_len_byte_array` data is interpreted as a sequence of elements of
+the same primitive data type.
```

Comment:
If this is only for primitive types, should the type be added to the `FixedSizedListType` struct?

Reply:
Added.